static/src/stylesheets/base/_links.scss

```diff
@@ -21,3 +21,10 @@ a[name] {
     display: inline-block;
```

Comment:
Hey @Calanthe is this hack in use currently? I had a quick look and couldn't find where this would apply